lib/purescript-cst/src/Language/PureScript/Environment.hs

```diff
@@ -52,6 +52,9 @@ data TypeClassData = TypeClassData
   { typeClassArguments :: [(Text, Maybe SourceType)]
   -- ^ A list of type argument names, and their kinds, where kind annotations
   -- were provided.
+  , typeClassVtaTypeVars :: [VtaTypeVar]
```

Comment:
Same question here regarding the separate spine vs adding it to `typeClassArguments`.